sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTableCatalog.scala

```diff
@@ -123,8 +123,14 @@ class JDBCTableCatalog extends TableCatalog with Logging {
         "ignored: " + properties.asScala.map { case (k, v) => s"$k=$v" }.mkString("[", ", ", "]"))
```

Comment:
I haven't removed the Warning yet. The `properties` here could be `provider`, `owner`, `location`, `comment` or `TBLPROPERTIES` key value pairs. I only implemented `comment`. Seems to me these are not options in standard database CREATE TABLE syntax. I am not sure if we need to support all of them.

Reply:
I think a more reasonable behavior is to fail if unsupported table properties are given. JDBC V2 is new in this release so we still have a chance to change it.

For these 4 builtin properties, I think we only need to support `comment`. We should ignore `provider` for now. And after the CRETE TABLE unification,  we can force users to not specify `USING` when creating table in JDBC v2.

Reply:
OK, so we will support `comment` and ignore `provider`, `owner`, and `location` for now.

For supported table properties, do we need to make them database specific? for example, `ENGINE` and `CHARSET` are supported table properties for MySQL but not supported by other databases.

Reply:
Instead of having a complete list of supported table properties for each of the databases, it might be better to send whatever table properties without checking and let it fail at database side if unsupported.

Reply:
yea, that sounds better, although it needs more work to update every dialect.